### DISCUSSION TITLE
link to a youtube account

### DISCUSSION THREAD

**Discussion by roromainlcs** [FreeTubeApp/FreeTube#1901] [Q&A]
Hi, i just found this project and i really like it, i'm considering to never use youtube again but i have a problem, i still want to be able to like dislike or comment on videos, so is there a way to link a youtube account to still be able to do that. I know it sounds dumb because it's the hole point of this project to not have google watching over, but i'd rather have the choice. I feel like this is what this type of project is about "having the choice" and i'd like to support my favourite youtubers or comment on a video without having to use the youtube website.

Anyway thank you for this amazing project.
Keep it up, love <3.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi u actually already answered ur own question. We wont implement that because it would defeat the main purpose of this application in the first place.

**Comment by ghost** [comment]
I know it's hard to implement but it would be cool though!
Doing that will requires redesigning FreeTube upon this.
History, bookmarks, subs should be sync and so on...
Basiclly implementing the whole Youtube _**in a different way**_ which just a mess...
